libwebsocketd/process_endpoint.go

```diff
@@ -13,22 +13,22 @@ import (
 )
 
 type ProcessEndpoint struct {
```

Comment:
Someday we should go thru and write docstrings to public methods, make this look pretty: http://godoc.org/github.com/joewalnes/websocketd/libwebsocketd